src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -1513,11 +1513,13 @@ public void CancelledBuildWithUnexecutedSubmission()
             _buildManager.EndBuild();
         }
 
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
Always nice to explain why:

```suggestion
#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously: needs to be async for xunit's timeout system
```